cogeo_mosaic/backends/sqlite.py

```diff
@@ -118,6 +122,11 @@ def write(self, overwrite: bool = False):
             self.delete()
 
         with self.db:
+            logger.debug(
+                f"Setting user_version to '{MOSAIC_JSON_VERSION}' in {self.db_path}."
+            )
+            self.db.execute(f"PRAGMA user_version = {MOSAIC_JSON_VERSION};")
```

Comment:
we can't `update` a mosaic created using the old mosaicjson specification.

In theory we could but then when we update again the specification we might regret this 😅